qa/wildfly/build.gradle

```diff
@@ -79,6 +79,7 @@ tasks.register("integTest", Test) {
   outputs.doNotCacheIf('Build cache is disabled for Docker tests') { true }
   maxParallelForks = '1'
   include '**/*IT.class'
+  systemProperty 'tests.security.manager', 'false'
```

Comment:
Him, why is this needed? It's not a very realistic scenario since we require security manager outside of tests. This flag is really only meant for special cases like the evil-tests.

Reply:
This was mentioned by @jasontedor in https://github.com/elastic/elasticsearch/issues/32572#issuecomment-410911823. I presume this is just because these tests are directly creating an HttpClient to hit a servlet vs using some other code path which is granted in our security policy (like the rest client itself).

Reply:
Ah I see, since we were previously not extending ESTestCase, we were not triggering the test security manager to be installed.